src/constants/commonStyles.js

```diff
@@ -1,7 +1,9 @@
-import { StyleSheet } from 'react-native';
+import { StyleSheet, Dimensions } from 'react-native';
 import typography from './typography';
 import { BLACK, ERROR, TRANSPARENT, BACKGROUND } from './colors';
 
+const SCREEN_HEIGHT = Dimensions.get('window').height;
```

Comment:
here now you can use the constant ```SCREEN_HEIGHT``` defined in ```common.js```

Reply:
Agree with @ManuViola77 comment